<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMainWindow</class>
 <widget class="QMainWindow" name="WMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioBooQer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QTreeView" name="chaptersView">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditKeyPressed</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="rootIsDecorated">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="goButton">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="WAudioPlayer" name="playerWidget" native="true"/>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <item row="0" column="0" colspan="2">
            <widget class="QCheckBox" name="renameCheck">
             <property name="text">
              <string>Rename processed files</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QCheckBox" name="chapterNoCheck">
             <property name="text">
              <string>Insert chapter number</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QCheckBox" name="autoChapterCheck">
             <property name="text">
              <string>Auto chapter name</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>First chapter number:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="firstChapterSpin">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Digits of chapter numbers:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="numberWidthSpin">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Language:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="languageCombo"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Threads:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QSpinBox" name="threadSpin">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>256</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="WAudioFormat" name="formatWidget" native="true"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openDirAction"/>
    <addaction name="separator"/>
    <addaction name="bindBookAction"/>
    <addaction name="separator"/>
    <addaction name="editTagAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="newChapterAction"/>
    <addaction name="dissolveChapterAction"/>
   </widget>
   <widget class="QMenu" name="menu_Playback">
    <property name="title">
     <string>&amp;Playback</string>
    </property>
    <addaction name="previousAction"/>
    <addaction name="stopAction"/>
    <addaction name="playAction"/>
    <addaction name="nextAction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Playback"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="openDirAction">
   <property name="text">
    <string>&amp;Open directory...</string>
   </property>
  </action>
  <action name="newChapterAction">
   <property name="text">
    <string>Create &amp;new chapter</string>
   </property>
   <property name="toolTip">
    <string>Create new chapter</string>
   </property>
  </action>
  <action name="previousAction">
   <property name="text">
    <string>Pre&amp;vious</string>
   </property>
  </action>
  <action name="stopAction">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="playAction">
   <property name="text">
    <string>&amp;Play</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="text">
    <string>Ne&amp;xt</string>
   </property>
  </action>
  <action name="dissolveChapterAction">
   <property name="text">
    <string>&amp;Dissolve last chapter</string>
   </property>
  </action>
  <action name="editTagAction">
   <property name="text">
    <string>Edit &amp;tag...</string>
   </property>
  </action>
  <action name="bindBookAction">
   <property name="text">
    <string>&amp;Bind book...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WAudioPlayer</class>
   <extends>QWidget</extends>
   <header>waudioplayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WAudioFormat</class>
   <extends>QWidget</extends>
   <header>WAudioFormat.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chaptersView</tabstop>
  <tabstop>goButton</tabstop>
  <tabstop>renameCheck</tabstop>
  <tabstop>chapterNoCheck</tabstop>
  <tabstop>autoChapterCheck</tabstop>
  <tabstop>firstChapterSpin</tabstop>
  <tabstop>numberWidthSpin</tabstop>
  <tabstop>languageCombo</tabstop>
  <tabstop>threadSpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
